src/types/interfaces/IAnime.ts

```diff
@@ -0,0 +1,9 @@
+export interface IAnime {
+    id: number;
+    title: string;
+    imageUrl: string;
+    genres: Array<string>;
+    yearStart: number | string | null; // год начала выпуска
+    yearEnd?: number | string | null; // год окончания выпуска
```

Comment:
null точно нужен? мб:
```suggestion
    yearEnd?: number | string ; // год окончания выпуска
```

Reply:
исправил